### PR TITLE
[DO NOT REVIEW] manually generating configs for Bazel 1.0.0

### PR DESCRIPTION
**Pull request by nlopezgi**
Also update Bazelisk version used in rbe_autoconfig

### CODE REVIEW

File: configs/ubuntu16_04_clang/versions.bzl [hunk 1]
```diff
@@ -3,8 +3,8 @@
...
 toolchain_config_spec1 = struct(config_repos = [], create_cc_configs = True, create_java_configs = True, env = {"ABI_LIBC_VERSION": "glibc_2.19", "ABI_VERSION": "clang", "BAZEL_COMPILER": "clang", "BAZEL_HOST_SYSTEM": "i686-unknown-linux-gnu", "BAZEL_TARGET_CPU": "k8", "BAZEL_TARGET_LIBC": "glibc_2.19", "BAZEL_TARGET_SYSTEM": "x86_64-unknown-linux-gnu", "CC": "clang", "CC_TOOLCHAIN_NAME": "linux_gnu_x86"}, java_home = "/usr/lib/jvm/java-8-openjdk-amd64", name = "8.0.0")
-_TOOLCHAIN_CONFIG_SPECS = [toolchain_config_spec0, toolchain_config_spec1]
-_BAZEL_TO_CONFIG_SPEC_NAMES = {"0.20.0": ["8.0.0"], "0.21.0": ["8.0.0"], "0.22.0": ["8.0.0", "9.0.0"], "0.23.0": ["8.0.0", "9.0.0"], "0.23.1": ["8.0.0", "9.0.0"], "0.23.2": ["9.0.0"], "0.24.0": ["9.0.0"], "0.24.1": ["9.0.0"], "0.25.0": ["9.0.0"], "0.25.1": ["9.0.0"], "0.25.2": ["9.0.0"], "0.26.0": ["9.0.0"], "0.26.1": ["9.0.0"], "0.27.0": ["9.0.0"], "0.27.1": ["9.0.0"], "0.28.0": ["9.0.0"], "0.28.1": ["9.0.0"], "0.29.0": ["9.0.0"], "0.29.1": ["9.0.0"]}
+_TOOLCHAIN_CONFIG_SPECS = [toolchain_config_spec0,toolchain_config_spec1]
```
Comment (smukherj1) [inline configs/ubuntu16_04_clang/versions.bzl]: Need to run buildifier on this file.

Comment (nlopezgi) [reply configs/ubuntu16_04_clang/versions.bzl]: done

**Pull request closed without merging**